mongodb/stitch-ios-sdk
# DOCSP-4384 Add analytics snippet to jazzy docs

- As in other sdks, when the argument "analytics" is passed to
generate_docs.sh, then the analytics snippet will be added.
- Jazzy base theme "fullwidth" is copied to contrib/docs-theme/
and the analytics snippet is added there.
- Local builds of docs (i.e. without passing "analytics" argument)
will produce docs without analytics, using the standard fullwidth
theme.

Staged:
https://docs-mongodbcom-staging.corp.mongodb.com/stitch/bush/docsp-4383-analytics/sdk/swift/index.html

Note: to verify,
- view source and see analytics snippet in the header, something like
```    
    <script>
        !function(){var analytics=window.analytics=window.analytics ........
```
- go to https://app.segment.com/mongodb/sources/docs/debugger and see incoming analytics from staging.
